paddle/fluid/inference/analysis/analyzer.h

```diff
@@ -66,7 +66,10 @@ class Analyzer : public OrderedRegistry<PassManager> {
   // merged in a larger fuse op. The small fusion will not break the pattern of
   // larger fusion.
   const std::vector<std::string> all_ir_passes_{{
-      // Manual update the passes here.
+// Manual update the passes here.
+#ifdef PADDLE_WITH_MKLDNN
+      "depthwise_conv_mkldnn_pass",  //
+#endif
```

Comment:
Could line 71 put together with line 85?

Reply:
It has to be run before convolution fuses so before line 82. This is why I put it up here at the beginning.

Reply:
GOT it, you can add this comment into codes.

Reply:
I will do it.